package circuitd;

message CreateCircuit {
  required uint64 client_id = 1;
  required uint32 next_hop_ip = 2;
  required uint64 next_hop_authenticator = 3;
}

message CircuitCreated {
  required CreateCircuit request = 1;
  required int64 authenticator = 2;
}
